src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -2207,8 +2214,7 @@ namespace HVACUnitaryBypassVAV {
                                                                                 CBVAV(CBVAVNum).CoolCoilCompIndex,
                                                                                 DataHVACGlobals::ContFanCycCoil,
                                                                                 HXUnitOn);
-                        } else if (DataLoopNode::Node(CBVAV(CBVAVNum).DXCoilOutletNode).Temp < CBVAV(CBVAVNum).CoilTempSetPoint &&
-                                   DataLoopNode::Node(CBVAV(CBVAVNum).DXCoilInletNode).Temp > CBVAV(CBVAVNum).CoilTempSetPoint) {
+                        } else if (DataLoopNode::Node(CBVAV(CBVAVNum).DXCoilOutletNode).Temp < CBVAV(CBVAVNum).CoilTempSetPoint) {
```

Comment:
Cleaning up unnecessary logic, inlet node T must be greater than CoilTempSP here, otherwise it's caught in the IF above this line